requirements.txt

```diff
@@ -3,17 +3,17 @@ setuptools>34.3.0
 wheel
 twine
 flake8
-nose
 nose-progressive
-coverage
-mock
 pytz
-mypy==0.770
-lxml # needed by mypy HTML coverage reporting
 pylint
-# sqlalchemy-vertica-python==0.4.5
-sqlalchemy-stubs>=0.3
+
 # this is pre-release, but seems to work!  Let's keep it pinned to a
 # commit until they make a release, just in case they break
 # compatibility.  Feel free to try to update!
+#
+# Note: it looks like numpy 1.20 is likely to roll these in natively
+# without a need for a separate stub package:
+#
+#  https://github.com/numpy/numpy/blob/master/doc/release/upcoming_changes/16558.new_feature.rst
+#  https://github.com/numpy/numpy/pull/16558
 git+https://github.com/numpy/numpy-stubs.git@be9a479#egg=numpy-stubs
```

Comment:
Unfortunately, I wasn't able to pull this one in to an extra, as there's no published package.  But it sounds like the next numpy will come with stubs by default!